src/js/common/components/NavButtons.jsx

```diff
@@ -81,7 +81,7 @@ export default class NavButtons extends React.Component {
     );
 
     let buttons;
-    if (path.endsWith('review-and-submit')) {
+    if (path.match(/review-and-submit(\/)?$/)) {
```

Comment:
Thanks @cvalarida, I hadn't compared the performance, but would it make more sense to include all of the potential code if doing so? Let me know if you have more ideas for speeding things up. And thanks for the code review.
<img width="977" alt="screen shot 2017-05-08 at 4 18 29 pm" src="https://cloud.githubusercontent.com/assets/16051172/25829349/4f5b39a4-340a-11e7-9d15-4f6dc624aeb6.png">